### DISCUSSION TITLE
Add telescope live_grep_args mappings

### DISCUSSION THREAD

**Discussion by kapral18** [LazyVim/LazyVim#1807] [Q&A] [answered]
How can I configure telescope live_grep_args like here https://github.com/nvim-telescope/telescope-live-grep-args.nvim?tab=readme-ov-file#configuration in lazyvim?

**Comment by dpetka2001** [comment, marked as answer]
Just install it like the instructions say in the [repo](https://github.com/nvim-telescope/telescope-live-grep-args.nvim?tab=readme-ov-file#installation) for `Lazy`. In the `opts` set the plugin's options and then also add the setup to telescope's `config` function. You can take a look at my [telescope](https://github.com/dpetka2001/dotfiles/blob/main/dot_config/nvim/lua/plugins/telescope.lua) config to see how I do it.

**Reply by dpetka2001** [reply]
Specifically the `project` telescope extension.

**Reply by kapral18** [reply]
Oh I missed that it's not config that's overwritten in lazyvim but opts. Thanks
